src/m_config.c

```diff
@@ -824,6 +824,11 @@ static default_t extra_defaults_list[] =
     CONFIG_VARIABLE_INT_HEX(opl_io_port),
 
     //!
+    // OPL chip type.
+	//
+    CONFIG_VARIABLE_INT(opl_type),
```

Comment:
It's good that we have a variable to control OPL chip type but it would be better if we reproduced the DMXOPTIONS variable that Vanilla Doom uses. Then we can have a config file variable (eg. "dmx_options") that sets this.